scheduler.py

```diff
@@ -2,23 +2,32 @@
 import queue
 import signal
 import sys
-import socket
+import random
+from Data.task import Task
+from Data.future import Future
+from worker_client import WorkerClient
+from logging_provider import logging
 
 class Scheduler:
-    def __init__(self, host, port):
+    def __init__(self):
         self.task_queue = queue.Queue()
         self.completion_status = {}
         self.lock = threading.Lock()
-        self.host = host
-        self.port = port
+        self.workers = []
         signal.signal(signal.SIGINT, self.sigterm_handler)
-        threading.Thread(target=self.start_server, daemon=True).start()
 
-    def submit_task(self, task):
-        with self.lock:
-            self.task_queue.put(task)
-            self.completion_status[task] = False
-        print("Task submitted to scheduler:", task)
+    def submit_task(self, task : Task) -> Future:
+        random_worker = random.choice(self.workers)
+        worker_client = WorkerClient(random_worker.split(":")[0], int(random_worker.split(":")[1]))
```

Comment:
do we need to remove the task from queue after submitting? A lock might be needed too?

Reply:
Since its a push model, i didn't include a task queue to maintain tasks. But we may need it for FT which can be added in subsequent PRs.